File: app/src/main/java/by/egorgutko/autorization/data/RoomDB/TaskDao.java
package by.egorgutko.autorization.data.RoomDB;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import java.util.List;

@Dao
public interface TaskDao  {

    @Insert
    void setTaskOfUSer(Person task);

    @Query("SELECT textForTask  FROM  persontask WHERE username = :name")
    List<String> getPersons(String name);



}

File: app/src/main/java/by/egorgutko/autorization/data/RoomDB/TaskDao.kt
package by.egorgutko.autorization.data.RoomDB

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query

@Dao
interface TaskDao {

    @Insert
    fun setTaskOfUSer(task: Person)

    @Query("SELECT textForTask  FROM  persontask WHERE username = :name")
    fun getPersons(name: String): List<String>


}
